src/validate.js

```diff
@@ -116,6 +120,12 @@ async function validator(options) {
     }
     return null;
   };
+
+  // Add a utility function that can be used to get all of the
+  // schemas that have been loaded.
+  validate.schemas = _.fromPairs(_.map(schemas, (s) => [s.name, s.content]));
```

Comment:
I think an array of schemas is enough... it's confusing with both name and schema.id...

I know id won't work as well initially, as the id will be schemas.t.n/..../....json. but later we can fix the schemaPrefix to be what the docs site will publish it under....